stratum/CMakeLists.txt

```diff
@@ -424,13 +430,15 @@ elseif(DPDK_TARGET)
     set_property(TARGET target_utils
                  PROPERTY IMPORTED_LOCATION ${LIBTARGET_UTILS})
 
+    add_dependencies(stratum krnlmon)
```

Comment:
1. You're adding the dependency on krnlmon unconditionally. What happens if `WITH_KRNLMON` is false?
2. Is it necessary to specify the dependency explicitly? From my reading, it looks as if making it a link library does this automatically.

Reply:
Added the check and you are right about 2nd. Don't need to specify the dependency explicitly